src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -38,16 +39,11 @@ export default View.extend({
       links = links.filter(l => $.isPlainObject(l));
     }
 
+    const settings = this.options.settings;
```

Comment:
if we are not using this new variable at many places then i would keep `this.options.settings` and not create a new variable